src/Page/Article.elm

```diff
@@ -151,6 +151,7 @@ viewAddComment postingDisabled maybeUser =
                         [ class "form-control"
                         , placeholder "Write a comment..."
                         , attribute "rows" "3"
+                        , value commentText
```

Comment:
This introduces a subtle bug—I just tried this same fix locally (before checking to see if someone else had already submitted a PR).

Put some text in the box, then move your cursor back to the beginning (or middle) and start mashing on the keyboard. The caret will at some point jump to the end of the text. See https://github.com/elm-lang/html/issues/55. 🙁

Reply:
This has been fixed in Elm 0.19. I'll merge this once that's released! 🎉